modules/transcription-service-ibm-watson-impl/src/main/java/org/opencastproject/transcription/ibmwatson/IBMWatsonTranscriptionService.java

```diff
@@ -471,6 +478,7 @@ protected String process(Job job) throws Exception {
     return result;
   }
 
+  // CHECKSTYLE:OFF
```

Comment:
Can we add why we turn this off?
Something like (not tested):

```suggestion
  // CHECKSTYLE:OFF  Example URL is too long but needs to stay like this
```

Reply:
In the default configuration, there can't be anything after `CHECKSTYLE:OFF`. One could configure that, but for now I just added a comment above it. Should be good for now.